src/services/order_watcher_service.ts

```diff
@@ -129,18 +143,12 @@ export class OrderWatcherService {
                 // so we need to leave space for the attributes on the model represented
                 // as SQL variables in the "AS" syntax. We leave 99 free for the
                 // signedOrders model
-                await this._connection.getRepository(SignedOrderEntity).save(signedOrdersModel, { chunk: 900 });
+                await this._connection.manager.save(signedOrdersModel, { chunk: DB_ORDERS_UPDATE_CHUNK_SIZE });
```

Comment:
I think we should use the `.getRepository` access pattern. I noticed a bug where sometimes it would save to PersistentSignedOrders and other times to regular SignedOrders, since the entities are so similar.